content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -551,7 +551,7 @@ const checkJson = await prisma.user.findMany({
   { "status": "expired", "insuranceID": 92 } // MySQL
   ```
 
-- If your array contains multiple objects, PostgreSQL will only return results if 
+- If your array contains multiple objects, PostgreSQL will only return results if
```

Comment:
This is still not a full sentence though, right?

Reply:
@janpio Right, this sentence seems incomplete. Can you ask someone on your team fix this? I don't know what the full sentence should look like.

Reply:
```suggestion
- If your filter array contains multiple objects, PostgreSQL will only return results if _all_ objects are present - not if at least one object is present.
```